src/lib/sql/sql_translator.cpp

```diff
@@ -1309,27 +1311,53 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_create_table(const hs
 
       // TODO(anybody) SQLParser is missing support for Hyrise's other types
       switch (parser_column_definition->type.data_type) {
+        case hsql::DataType::SMALLINT:
+          std::cout << "WARNING: Implicitly converting SMALLINT to INT\n";
+          [[fallthrough]];
         case hsql::DataType::INT:
           column_definition.data_type = DataType::Int;
           break;
         case hsql::DataType::LONG:
           column_definition.data_type = DataType::Long;
           break;
+        case hsql::DataType::DECIMAL:
+          std::cout << "WARNING: Implicitly converting DECIMAL to FLOAT\n";
+          column_definition.data_type = DataType::Float;
+          break;
+        case hsql::DataType::REAL:
+          std::cout << "WARNING: Implicitly converting REAL to FLOAT\n";
+          column_definition.data_type = DataType::Float;
+          break;
         case hsql::DataType::FLOAT:
           column_definition.data_type = DataType::Float;
           break;
         case hsql::DataType::DOUBLE:
           column_definition.data_type = DataType::Double;
           break;
         case hsql::DataType::CHAR:
+          std::cout << "WARNING: Ignoring the length of CHAR as Hyrise has no way of working with it\n";
```

Comment:
"Ignoring the length of CHAR. Hyrise's strings are not length-limited." ?

Reply:
I don't think the "no way of working with it" is not clear enough as a user information. Sounds like we don't know what to do with the text.